src/pal/src/configure.cmake

```diff
@@ -366,6 +366,9 @@ check_cxx_source_runs("
 #include <stdlib.h>
 #include <time.h>
 #include <sys/time.h>
+#ifdef (__linux__)
```

Comment:
@myungjoo You are right, I've missed the fact that it is not defined for the configure.cmake, so the `__linux__` usage over there is correct.